### ISSUE TITLE
skull hit detection / sprite animation broken on device

### ISSUE THREAD

**Issue by makapuf** [makapuf/bitbox-alterego#5]
skull-hit detection and sprite animation don't appear to be working on the bitbox.  (works fine on emulator).

**Comment by makapuf** [comment]
ok this one was tricky.
- Why were collision and sprite frame animations disabled ? 
  Because they only happened on every frame, and that frame was never happening since game_frame happened on odd frames (vga_frame was skipping one frame every frame)
- why was it skipping one frame every frame ? because it did not have enough time to process. Why ? because graph_frame, which is higher priority, was not letting it.
- why was graph_frame long ? Because sprite display frame callback for 8 bits was taking too long, because an optimization disabled. Re-enabling it in lib/blitter_sprites.c did the trick ! 

TL;DR upgrade your kernel

**Comment by lowagner** [comment]
great, thanks!  nice work.  will test later.

**Comment by lowagner** [comment]
still not working -- but it doesn't look like you pushed your lib/blitter changes to the SDK.
